doc/gtsam.lyx

```diff
@@ -1937,7 +1948,7 @@ reference "fig:CompareMarginals"
 \end_inset
 
 , where I show the marginals on position as covariance ellipses that contain
- 68.26% of all probability mass.
+ 99.9996% of all probability mass.
```

Comment:
Would be helpful to specify that these are "5-sigma", e.g.

`, where I show the marginals on position as 5-sigma covariance ellipses that contain
 99.9996% of all probability mass.`